test/ut_expectations/test_expec_not_to_be_null.pkb

```diff
@@ -0,0 +1,175 @@
+create or replace package body test_expec_not_to_be_null
+is
+
+    procedure execute_expectation(a_data_type in varchar2,
+                                    a_data_value in varchar2)
+    is
+        l_execute varchar2(32000);
+    begin
+         -- arrange
+        l_execute := '  declare
+                            l_expected '||a_data_type||' := '||a_data_value||';
+                        begin
+                            --act - execute the expectation
+                            ut3.ut.expect(l_expected).not_to_be_null();
+                        end;';
+
+        execute immediate l_execute;
+    end;
+
+    procedure test_success_expectacion(a_data_type in varchar2,
+                                    a_data_value in varchar2)
+    is
+    begin
+        execute_expectation(a_data_type, a_data_value);
+
+        --assert - check that expectation was executed successfully
+        ut3_latest_release.ut.expect(ut3.ut_expectation_processor.get_status()).to_equal(ut3.ut_utils.tr_success);
```

Comment:
Please remove reference to `ut3_latest_release`
We use utPLSQL v3 (latest release) to test our software (utPLSQL in ut3 schema).
In  **development mode** we do not have synonyms for `ut3` schema.
The synonyms are created to the `ut3_latest_release`